lib/fluent/plugin/in_systemd.rb

```diff
@@ -13,68 +13,65 @@ class SystemdInput < Input
     config_param :strip_underscores, :bool, default: false
     config_param :tag, :string
 
-    attr_reader :tag
+    attr_reader :tag, :journal, :running, :pos_writer
```

Comment:
These `attr_readers` are not needed if we are using instance variables to access the values...